app/lib/dynamicRequire.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
   'views/user/IdentifyView': function () { return import(/* webpackChunkName: "user" */ 'views/user/IdentifyView') },
   'views/account/IsraelSignupView': function () { return import(/* webpackChunkName: "account" */ 'views/account/IsraelSignupView') },
   'views/LegalView': function () { return import(/* webpackChunkName: "LegalView" */ 'views/LegalView') },
+  'views/play/VoyagerUnitMap': function() { return import(/*webpackChunkName: "play" */ 'views/play/VoyagerUnitMap') },
```

Comment:
Should we, and could we start a separate ozaria webpack chunk for the new ozaria pieces?

Thus as we make infrastructure and front end improvements for ozaria. We won't be blowing up the size of the play chunk for the code combat players.
And we might be able to make the ozaria chunk smaller as well?

Reply:
Thats a good point, I am also going to rename voyager to ozaria at all places. How about updating this chunk name to be `ozariaPlay`?